spec/features/assets/asset_submission_spec.rb

```diff
@@ -15,6 +15,34 @@
     create(request_factory, study: study, project: project, asset: asset, request_type: original_request_type)
   end
 
+  describe 'The request form does not set default values' do
```

Comment:
I think also add tests on submit to check the 3 fields must be set with values to continue, or error in GUI if not.
The story is saying they don't want default values set because people are not checking and submitting with the defaults when not appropriate, but that the user must set values.